fluentui_tablayout/src/main/java/com/microsoft/fluentui/tokenized/segmentedcontrols/Pill.kt

```diff
@@ -66,14 +66,16 @@ data class PillMetaData(
  * @param style Color Scheme of pill shaped button. Default: [FluentStyle.Neutral]
  * @param interactionSource Interaction Source Object to handle gestures.
  * @param pillButtonTokens Tokens to customize the design of pill button.
+ * @param positionNumber Pill button number when inside PillBar. Default: null
  */
 @Composable
 fun PillButton(
     pillMetaData: PillMetaData,
     modifier: Modifier = Modifier,
     style: FluentStyle = FluentStyle.Neutral,
     interactionSource: MutableInteractionSource = remember { MutableInteractionSource() },
-    pillButtonTokens: PillButtonTokens? = null
+    pillButtonTokens: PillButtonTokens? = null,
+    positionNumber: Int? = null
```

Comment:
Lets not add additional parameter for accessibility here. This parameter might not have any usage other than accessibility.

Reply:
Removing this, and shifting the control to PillBar